src/components/Dashboard/Pay/index.tsx

```diff
@@ -67,16 +67,10 @@ export default function Pay() {
           </Button>
         </Tooltip>
       )
-    } else if (
-      fcMetadata.reservedRate === 200 ||
-      fcMetadata.payIsPaused ||
-      overridePayDisabled
-    ) {
+    } else if (fcMetadata.payIsPaused || overridePayDisabled) {
```

Comment:
**issue** I think this check needs to be based on the contract version. We still want it disabled for v1 iiuc.

Reply:
yep this is true. the check should basically be `disabled = (reservedRate == 200 && terminal.version == 1.1)`

Reply:
Oh yep misread this one! Fixed up, thanks